pom.xml

```diff
@@ -292,7 +292,7 @@
     <!-- Version used in Connect -->
     <connect.guava.version>32.0.1-jre</connect.guava.version>
     <guava.failureaccess.version>1.0.1</guava.failureaccess.version>
-    <io.grpc.version>1.56.0</io.grpc.version>
+    <io.grpc.version>1.59.0</io.grpc.version>
```

Comment:
This is `1.59.0` instead of `1.59.3` because it's their convention.
- https://repo1.maven.org/maven2/io/grpc/grpc-protobuf/